oar/core/worksheet.py

```diff
@@ -158,7 +158,7 @@ def update_shipment_info(self, shipment):
         Args:
             shipment (str): shipment info of current release
         """
-        self._ws.update_acell(LABEL_SHIPMENT, shipment)
+        self._ws.update_acell(LABEL_SHIPMENT, self._to_hyperlink(shipment, shipment))
```

Comment:
What if the `shipment` parameter will contain two or more urls? The hyperlink won't work as expected.

Reply:
It’s not good practice to rely on an input format that doesn’t match the function’s expectations. Passing multiple values as a single string can lead to unexpected behavior.

This change creates a potential bug that was not present before.